src/js/HuesCanvas.js

```diff
@@ -58,14 +58,15 @@ class HuesCanvas {
         this.blurDistance = 0;
         this.xBlur = false;
         this.yBlur = false;
-        
```

Comment:
I will request that you keep the "whitespace is indented" style. I'll consider nuking it across the entire project in a future commit.